toolbox/toolbox-test.sh

```diff
@@ -103,9 +105,13 @@ if [ -z "$(govc find / -type m -name "$vm")" ] ; then
   govc datastore.upload config.iso "$vm/config.iso" >/dev/null
   device=$(govc device.cdrom.add -vm "$vm")
   govc device.cdrom.insert -vm "$vm" -device "$device" "$vm/config.iso"
+
+  vm_path="$(govc find / -type m -name "$vm")"
+else
+  govc object.collect -s "$vm_path" -guest.toolsStatus toolsNot* # wait for previous toolbox to unregister
```

Comment:
is this something we could check from the guest side to avoid accidentally running two toolboxes?